src/generic_ui/locales/en/messages.json

```diff
@@ -667,6 +667,14 @@
     "description": "Notification for when the user has been logged out of the social network.",
     "message": "You have been logged out of __network__"
   },
+  "PRE_LOG_OUT": {
+  "description": "Notification to ask user if they are sure that they want to log out.",
+  "message": "Are you sure you want to log out? You are granting access to __name__ and __numOthers__ others"
```

Comment:
To keep the language consistent with the rest of uProxy, replacing "You are granting access to" should be replaced with "You are sharing access with"
(Granted access for uProxy means that another user has received permission to proxy, sharing access with means that a user is currently proxying)